WebCryptoAPI/import_export/importKey_failures.js

```diff
@@ -162,8 +168,8 @@ function run_test(algorithmNames) {
     // Algorithms normalize okay, usages ok and valid key. The lack of the mandatory JWK parameter must throw a syntax error.
     testVectors.forEach(function(vector) {
         var name = vector.name;
-        missingJWKFieldKeyData.forEach(function(test) {
-            allAlgorithmSpecifiersFor(name).forEach(function(algorithm) {
+        allAlgorithmSpecifiersFor(name).forEach(function(algorithm) {
+           getMissingJWKFieldKeyData(algorithm).forEach(function(test) {
                 allValidUsages(validUsages(vector, 'jwk', test.data)).forEach(function(usages) {
                     [true, false].forEach(function(extractable) {
                         testError('jwk', algorithm, test.data, name, usages, extractable, "DataError", "Missing JWK '" + test.param + "' parameter");
```

Comment:
The comment says this must throw a syntax error, but this is specifying "DataError" here - am I missing something?